fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadMgr.java

```diff
@@ -546,14 +546,19 @@ public StreamLoadTask getTaskById(long id) {
     }
 
     // return all of stream load task named label in all of db
+    // return all tasks if label is null
     public List<StreamLoadTask> getTaskByName(String label) {
         List<StreamLoadTask> result = Lists.newArrayList();
         readLock();
         try {
-            for (Map<String, StreamLoadTask> labelToStreamLoadTask : dbToLabelToStreamLoadTask.values()) {
-                if (labelToStreamLoadTask.containsKey(label)) {
-                    result.add(labelToStreamLoadTask.get(label));
+            if (label != null) {
+                StreamLoadTask task = idToStreamLoadTask.get(label);
```

Comment:
1. what's the difference between `dbToLabelToStreamLoadTask` and `idToStreamLoadTask`? 
2. can we use `where label like 'abc%' to search? and `where db_name = 'xxx'` to filter?

Reply:
1. Both of them are map structure, but with different key/value, `idToStreamLoadTask`'s key is task id, and `dbToLabelToStreamLoadTask`'s key is db id
2. They should both be supported, but for now it looks both are missed, maybe we can try to fix them in this pr.